hera_pspec/tests/test_pspecdata.py

```diff
@@ -229,7 +230,23 @@ def test_get_G(self):
                 for i in range(Nfreq):
                     for j in range(Nfreq):
                         self.assertGreaterEqual(G[i,j], -min_diagonal*10**-10)
-            
+
+    def test_scalar(self):
+        dfiles = [
+            'zen.2458042.12552.xx.HH.uvXAA',
+            'zen.2458042.12552.xx.HH.uvXAA'
+        ]
+
+        d = []
+        for dfile in dfiles:
+            _d = uv.UVData()
+            _d.read_miriad(DATADIR + dfile)
+            d.append(_d)
+        w = [None for _d in dfiles]
+        self.ds = pspecdata.PSpecData(dsets=d, wgts=w, beam=self.bm)
+
+        scalar = self.ds.scalar()       
+        self.assertAlmostEqual(scalar,3732415176.85 / 10**9) 
```

Comment:
Can you make a note of where this number comes from too, please?

Reply:
Done!